frontend/container_init.sh

```diff
@@ -9,7 +9,7 @@ if [ ! -e ./node_modules ]; then
 fi
 ln -sf /opt/node_app/node_modules/* /opt/node_app/node_modules/.bin ./node_modules/.
 if [ ! -z "$API_URL" ]; then
-  cat src/configs/$DEPLOYMENT_STAGE.js | envsubst >src/configs/configs.js
```

Comment:
Didn't end up going with this solution because it assumes a lot of hardcoded values, most importantly API_URL